ui/apps/platform/src/Containers/Vulnerabilities/components/SnoozeCvesModal/useSnoozeCveModal.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+import { resourceTypes } from 'constants/entityTypes';
+
+type SnoozeableCveType = typeof resourceTypes.CLUSTER_CVE | typeof resourceTypes.NODE_CVE;
```

Comment:
Because this looks like the first use of `resourceTypes` in Vulnerability folder, what do you think about:
* string enumeration `type SnoozeableCveType = 'CLUSTER_CVE' | 'NODE_CVE';`
* import this type in SnoozeCvesModal
* just plain `'CLUSTER_CVE'` or `'NODE_CVE'` wherever either value is used?
    And leave the checking to TypeScript to borrow from Greyhound.